### ISSUE TITLE
Error Code 65 when building for iOS

### ISSUE THREAD

**Issue by NicholasEli** [Wizcorp/phonegap-facebook-plugin#1325]
I get the following error when trying to build for ios.
CompileC build/facebook.build/Debug-iphonesimulator/facebook.build/Objects-normal/i386/FacebookConnectPlugin.o facebook/Plugins/phonegap-facebook-plugin/FacebookConnectPlugin.m normal i386 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)

Error: Error code 65 for command: xcodebuild with args: -xcconfig,/Users/nicholas/Desktop/phonegap/facebook/platforms/ios/cordova/build-debug.xcconfig,-project,facebook.xcodeproj,ARCHS=i386,-target,facebook,-configuration,Debug,-sdk,iphonesimulator,build,VALID_ARCHS=i386,CONFIGURATION_BUILD_DIR=/Users/nicholas/Desktop/phonegap/facebook/platforms/ios/build/emulator,SHARED_PRECOMPS_DIR=/Users/nicholas/Desktop/phonegap/facebook/platforms/ios/build/sharedpch

**Comment by ghost** [comment]
Same problem, found any fix ?

**Comment by seriousjelly** [comment]
Any fix?

**Comment by ghost** [comment]
Yep, use this instead https://github.com/jeduan/cordova-plugin-facebook4
Then remove ios platform and add it again
